nameko_sqlalchemy.py

```diff
@@ -22,11 +22,14 @@ def setup(self):
             'service_name': service_name,
             'declarative_base_name': decl_base_name,
         })
+        self.engine = create_engine(self.db_uri)
+
+    def stop(self):
+        self.engine.dispose()
```

Comment:
That's a good idea. If it is too tricky I guess you could just mock stuff out and make sure you are disposing of the engine correctly. :)